### ISSUE TITLE
Update "A non-visible element gets the focus - is this a fail?"

### ISSUE THREAD

**Issue by Jefferydo** [cenw-wscoe/WCAG-in-Plain-Language#51]
> "If the invisible element gets on focus in the middle of an operation (tabing through elements in a menu for example): Yes. The focus order needs to be constent and predictable while tabbing through a section where the tab order is important and might interfere with the understanding of the section.
> 
> "However, if this occure outside or between two sections and it doesn't break the meaning of the content or the expected behaviour, no this is not an issue althought it is still not encouraged."

It's a failure of SC 2.4.7: Focus Visible, not 2.4.3: Focus order.
From Understanding 2.4.7: "It must be possible for a person to know which element among multiple elements has the keyboard focus"

Considering that it's a failure to remove the focus indicator from a control, missing the whole control is a bigger problem.

It's also frequently a failure of 4.1.2, when the mystery tab stop has no accessible name.

Here's a Deque page on the issue, [aria-hidden elements do not contain focusable elements](https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus)
